src/details/ArborX_DetailsDistributedTreeSpatial.hpp

```diff
@@ -52,12 +53,13 @@ DistributedTreeImpl::queryDispatch(SpatialPredicateTag, Tree const &tree,
   auto const &bottom_tree = tree._bottom_tree;
   auto comm = tree.getComm();
 
-  Kokkos::View<int *, MemorySpace> indices(
-      "ArborX::DistributedTree::query::spatial::indices", 0);
+  Kokkos::View<int *, MemorySpace> intersected_ranks(
+      "ArborX::DistributedTree::query::spatial::intersected_ranks", 0);
+  query(top_tree, space, predicates, LegacyDefaultCallback{}, intersected_ranks,
```

Comment:
```suggestion
  top_tree.query(space, predicates, LegacyDefaultCallback{}, intersected_ranks,
```
(not feeling too strongly about it)

Reply:
I kinda agree. Don't feel strongly either, but using free function is kind of weird. Not really part of this PR, but I'll change it as a drive-by.